x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1484,4 +1484,48 @@ export const tasks: TelemetryTask[] = [
       };
     },
   },
+  {
+    name: 'top_traces',
+    executor: async ({ indices, telemetryClient }) => {
+      const response = await telemetryClient.search({
+        index: indices.transaction,
+        body: {
+          size: 0,
+          track_total_hits: false,
+          timeout,
+          query: {
+            bool: {
+              filter: [range1d],
+            },
+          },
+          aggs: {
+            top_traces: {
+              terms: {
+                field: 'trace.id',
+                size: 100,
+              },
+            },
+            max: {
+              max_bucket: {
+                buckets_path: 'top_traces>_count',
```

Comment:
I haven't seen this syntax before, is it like a `.` notation? or just a literal?

Reply:
The `buckets_path` parameter is used to reference the result of another aggregation within the same query
The difference between `.` notation and `>` is that `top_traces>_count` reference a specific aggregation by its name and `top_traces._count` directly reference a field within an aggregation without specifying the aggregation's name.
I think in this case both notations will work. https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline.html

Reply:
Got it now! it's like working with css 🪄